src/tripCard/index.tsx

```diff
@@ -39,7 +39,7 @@ const StyledTripCard = styled(TripCard)`
     padding-top: 0;
   }
 
-  &.kirk-item--highlighted .kirk-text-body {
+  & .kirk-item--highlighted .kirk-text-body {
```

Comment:
is it intended?

Reply:
Yes, I messed up before and mistook this as the main "highlighted" class while it's a modifier for an item inside the card